### PR TITLE
Only send the client brand on game join

### PR DESCRIPTION
**Pull request by DoctorMacc**
Matches vanilla behavior and prevents BungeeCord from throwing a fit at random.

### CODE REVIEW

File: connector/src/main/java/org/geysermc/connector/utils/PluginMessageUtils.java [hunk 1]
```diff
@@ -23,39 +23,29 @@
...
-
-    @Override
-    public void translate(ServerPluginMessagePacket packet, GeyserSession session) {
-        if (packet.getChannel().equals("minecraft:brand")) {
-            session.sendDownstreamPacket(
-                    new ClientPluginMessagePacket(packet.getChannel(), brandData)
-            );
-        }
+    /**
+     * @return the brand information of the Geyser client
```
Comment (rtm516) [inline connector/src/main/java/org/geysermc/connector/utils/PluginMessageUtils.java:45]: Not super necessary but a description of this function would be good, so its a proper JavaDoc comment.
```
Get the prebuilt brand as a byte array
```

**Review by Tim203 — APPROVED** [review]

File: connector/src/main/java/org/geysermc/connector/utils/PluginMessageUtils.java [hunk 2]
```diff
@@ -85,4 +75,5 @@ private static int getVarIntLength(int number) {
         }
         return 5;
     }
+
```
Comment (rtm516) [inline connector/src/main/java/org/geysermc/connector/utils/PluginMessageUtils.java:78]: Un-needed, can this be removed to clean up the commit.

**Review by rtm516 — DISMISSED** [review]

**Comment by NovaPixell** [comment]
Excuse me guys, i had this issues some month ago. Has been fixed? Now can I use F3Name on Bungee without risking the user from minecraft bedrock crashes?

**Comment by Tim203** [comment]
The linked issue has been closed with no further reactions, so I expect that it has been fixed.

**Pull request merged**